app/src/components/landing/pricing/PricingIndexPage/OtherWaysToMakePurchase/RequestQuoteModal/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { generateSupportTicketNumber } from "utils/Misc";
+import { getFunctions, httpsCallable } from "firebase/functions";
+import { EVENTS } from "./analytics";
+import { trackRequestQuoteFormSubmitted } from "./analytics";
+
+const RequestQuoteModal = ({ isOpen, handleToggleModal }) => {
+  const [formState, setFormState] = useState("form"); // 'form', 'loading', 'success'
+
+  const handleSubmit = async (values) => {
+    trackRequestQuoteFormSubmitted();
+    setFormState("loading");
+    const salesInboundNotification = httpsCallable(getFunctions(), "salesInboundNotification");
+    try {
+      await salesInboundNotification({
+        notificationText: `${EVENTS.REQUEST_QUOTE_FORM_SUBMITTED} triggered with document type ${values.documentType}, purchase type ${values.purchaseType}, number of licenses ${values.numberOfLicenses}, subscription period ${values.subscriptionPeriod}, company name ${values.companyName}, and email ${values.email}`,
+      });
+    } catch (error) {
+      console.error(error);
+    }
+    setFormState("success");
+  };
+
+  const styles = {
+    formItem: {
+      marginBottom: "16px",
+    },
+    submitButton: {
+      width: "100%",
+    },
+    successMessage: {
+      textAlign: "center",
+    },
+    loader: {
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      height: "100%",
+    },
+  };
+
+  return (
+    <RQModal centered title="Request a Quote" open={isOpen} onCancel={handleToggleModal}>
+      <div className="rq-modal-content">
+        {formState === "form" && (
+          <Form layout="vertical" onFinish={handleSubmit}>
+            <Form.Item
+              name="documentType"
+              label="Type of Document"
+              rules={[{ required: true, message: "Please select the type of document" }]}
+              style={styles.formItem}
+            >
+              <Radio.Group>
+                <Radio value="quotation">Quotation/Estimate</Radio>
+                <Radio value="proforma">Proforma Invoice</Radio>
+              </Radio.Group>
+            </Form.Item>
+            <Form.Item
+              name="purchaseType"
+              label="Type of Purchase"
+              rules={[{ required: true, message: "Please select the type of purchase" }]}
+              style={styles.formItem}
+            >
+              <Radio.Group>
+                <Radio value="newLicense">New License</Radio>
+                <Radio value="licenseUpdate">License Update</Radio>
+                <Radio value="licenseRenewal">License Renewal</Radio>
+              </Radio.Group>
+            </Form.Item>
+            <Form.Item
+              name="numberOfLicenses"
+              label="Number of Licenses"
+              rules={[{ required: true, message: "Please enter the number of licenses" }]}
+              style={styles.formItem}
+            >
+              <Input type="number" min={1} />
```

Comment:
we have RQInput as well, are you sure that can't be used here (and everywhere else the antd Input component has directly been used)?